.travis.yml

```diff
@@ -30,35 +30,39 @@ env:
 
 matrix:
   include:
-  - rvm: 2.1
+  - rvm: 2.1.9
     sudo: true
+    dist: trusty
     script: sudo -E $(which bundle) exec rake spec;
     # also remove integration / external tests
     bundler_args: --without changelog development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
-  - rvm: 2.2
+  - rvm: 2.2.5
     sudo: true
+    dist: trusty
     script: sudo -E $(which bundle) exec rake spec;
     # also remove integration / external tests
     bundler_args: --without changelog development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
-  - rvm: 2.3.0
+  - rvm: 2.3.1
     sudo: true
+    dist: trusty
     script: sudo -E $(which bundle) exec rake spec;
     # also remove integration / external tests
     bundler_args: --without changelog development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
   - rvm: rbx
     sudo: true
+    dist: trusty
```

Comment:
can we move `dist: trusty` to the global part and then override later if we need precise?